jetty-util/src/main/java/org/eclipse/jetty/util/ssl/SslContextFactory.java

```diff
@@ -1953,4 +1987,56 @@ public X509 getX509()
             return _x509;
         }
     }
+
+    public static class Client extends SslContextFactory
+    {
+        public Client()
+        {
+            this(false);
+        }
+
+        public Client(boolean trustAll)
+        {
+            super(trustAll);
+        }
+
+        @Override
+        protected void checkConfiguration()
+        {
+            checkTrustAll();
+            checkEndPointIdentificationAlgorithm();
+            super.checkConfiguration();
+        }
+    }
+
+    public static class Server extends SslContextFactory
+    {
+        public Server()
+        {
+            setEndpointIdentificationAlgorithm(null);
+        }
+
+        @Override
+        public boolean getWantClientAuth()
+        {
+            return super.getWantClientAuth();
```

Comment:
these are also deprecated methods!  Would be better to directly access the field.

Reply:
It is not better to access the field. When merging to 10, I will take care of _moving_ the fields here.
So, for now, it stays like that, in 10 will be cleaned up properly.